packages/@lwc/errors/src/compiler/__tests__/error-info.spec.ts

```diff
@@ -0,0 +1,44 @@
+import * as errorInfo from '../error-info';
+
+// All exported objects are maps of label/error info, except for GENERIC_COMPILER_ERROR,
+// which is a top-level error info object
+const { GENERIC_COMPILER_ERROR, ...errors } = errorInfo;
+
+const errorInfoMatcher = {
+    code: expect.any(Number),
+    message: expect.any(String),
+    url: expect.any(String),
+    // Technically not *any* number, but jest doesn't have oneOf
+    level: expect.any(Number),
+};
+
+it('GENERIC_COMPILER_ERROR should be an error info object', () => {
+    expect(GENERIC_COMPILER_ERROR).toEqual(errorInfoMatcher);
+});
+
+describe.each(Object.entries(errors))('%s errors', (_key, map) => {
+    it('labels should all be UPPER_SNAKE_CASE', () => {
+        Object.keys(map).forEach((label) => {
+            expect(label).toMatch(/[A-Z](?:_[A-Z]+?)*?$/);
```

Comment:
I guess we're missing a `^` and a `+`:
```suggestion
            expect(label).toMatch(/^[A-Z]+(?:_[A-Z]+?)*?$/);
```
The following regex allows a trailing `_` but it's 9/16" shorter:
```suggestion
            expect(label).toMatch(/^[A-Z][A-Z_]*$/);
```